### ISSUE TITLE
en.wikipedia.org: breakage

### ISSUE THREAD

**Issue by sunlightspectre** [uBlockOrigin/uAssets#23861]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Search [here](https://github.com/uBlockOrigin/uAssets/issues) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL(s) where the issue occurs.

```yaml
https://en.wikipedia.org/wiki/Rhodonite?useskin=vector
https://en.wikipedia.org/wiki/Rhodonite?useskin=vector#/media/File:8224M-rhodonite3.jpg
```

### Description

So far as I am _aware_ this is an edge case interaction between uBlock 1.58.0 and wikipedia's classic 'vector' skin.  
For those of who use vector while working/editing wikipedia the issue can be pretty frustrating.  
The issue is as follows:

On interacting with inline images hosted on wikipedia - the second link provided -  
Those images normally redirect.  
When a new page with images is loaded, or an image is selected _once,_ the screen flashes black for roughly one second.  
The following message is displayed.

`Error loading MediaViewer: Failed dependency: mmv`

And then, on a second interaction, the image can be loaded as normal.  
Disabling uBlock on any vector page is enough to temporarily circumvent the issue.  
I am reporting it in case there are other potential edge cases affected by whatever is causing it.    
(And, of course, because it'd make my life more convenient if it were fixed!)

Browser - Waterfox G6.0.14 64-bit

### Other extensions used

Violentmonkey + https://greasyfork.org/en/scripts/458501-vector-layout-for-wikipedia
However, issue persists with scripts + specific script disabled, so long as vector is being used.  
This includes while logged in and vector selected as a preferred display.

### Configuration

<details>
<summary>Details</summary>

```yaml

uBlock Origin: 1.58.0
Firefox: 115
filterset (summary):
 network: 194562
 cosmetic: 156505
 scriptlet: 23859
 html: 1809
listset (total-discarded, last-updated):
 added:
  https://raw.githubusercontent.com/laylavish/uBlockOrigin-HUGE-AI-Blocklist/main/list.txt: 3755-33, 9h.57m
  https://secure.fanboy.co.nz/fanboy-annoyance_ubo.txt: 99193-48339, 9h.57m
  https://ublockorigin.github.io/uAssets/filters/resource-abuse.txt: 78-0, 9h.57m
  fanboy-cookiemonster: 67227-67204, 9h.57m
  fanboy-social: 16115-16112, 9h.57m
 default:
  user-filters: 245740-97859, never
  ublock-filters: 38383-149, 3d.1h.26m Δ
  ublock-badware: 8736-3, 3d.1h.26m Δ
  ublock-privacy: 920-82, 3d.1h.26m Δ
  ublock-unbreak: 2335-32, 3d.1h.26m Δ
  easylist: 87324-19757, 3d.1h.26m Δ
  easyprivacy: 50953-8589, 3d.1h.26m Δ
  urlhaus-1: 13835-1, 9h.57m
  plowe-0: 3726-3380, 3d.3h.27m
  ublock-quick-fixes: 163-12, 4h.57m Δ
filterset (user): [array of 235078 redacted]
trustedset:
 added: [array of 65 redacted]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 19078 ms
 maxAssetCacheWait: 1068 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 1
 network:
  wikipedia.org: 1
 extended:
  ###centralNotice
  ###siteNotice

```
</details>

**Comment by stephenhawk8054** [comment]
I'm still not clear how to follow your steps to reproduce. Can you record video how to do it?

I accessed `https://en.wikipedia.org/wiki/Rhodonite?useskin=vector#/media/File:8224M-rhodonite3.jpg` already and it appears like this to me

![image](https://github.com/uBlockOrigin/uAssets/assets/66517106/ba3d473b-a252-4527-8de7-b8f5f3a6fb38)

What should I do next?

**Comment by sunlightspectre** [comment]
Hmn. That's weird.  
I thought I tested it pretty thoroughly.  
Sure, let me see what I can do. [Hope it's okay I use the same links. ](https://i.imgur.com/ufLDKrF.mp4)

But if you can't duplicate it, there's not much else I can do to help duplicate it unfortunately.  
For now the issue is 'solved' by me simply disabling uBlock on wikipedia which is fine in the first place.  
I do feel it's unfortunate I can't think of a good next step to repro.  
Well, I reported it so if anyone else has similar, hopefully I documented it well enough to troubleshoot further.

**Comment by stephenhawk8054** [comment]
Are you logged in to the site?

**Comment by stephenhawk8054** [comment]
I just realized that you are using a heavily modified configuration. Test again in a new browser profile with only uBO and its default settings.

**Comment by sunlightspectre** [comment]
> Are you logged in to the site?

As of roughly five minutes ago, being _logged in_ works again.  
But most of my reading is done _logged out,_ as I was above in the video sample.  

> I just realized that you are using a heavily modified configuration. Test again in a new browser profile with only uBO and its default settings.

Sorry, I thought I made it clear in the report.  
I've also tried on a default install of Waterfox, same problem.  
_However_ I'm beginning to suspect it's an edge case to di with redirection, and not uBO.  
Going to close this and see if I can't figure out what, or why, as the problem stared after I updated to 1.58.  
Thanks for the time and assists!
